arduino/libraries/librariesresolver/cpp.go

```diff
@@ -53,6 +54,49 @@ func (resolver *Cpp) ScanFromLibrariesManager(lm *librariesmanager.LibrariesMana
 	return nil
 }
 
+// ScanIDEBuiltinLibraries reads ide-builtin librariers loaded in the LibrariesManager to find
```

Comment:
```suggestion
// ScanIDEBuiltinLibraries reads IDE-builtin libraries loaded in the LibrariesManager to find
```
Fix typo in doc comment.